pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>services/private_ip_manager</module>
         <module>services/port_manager</module>
         <module>services/node_manager</module>
+        <module>services/data_plane_manager</module>
```

Comment:
Add new plugins to this pom.xml file (ref: https://github.com/futurewei-cloud/alcor/pull/241/files#diff-9fe0ed938fde38c66436944093832b38)

            <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>0.8.5</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>prepare-agent</goal>
                         </goals>
                     </execution>
                     <!-- attached to Maven test phase -->
                     <execution>
                         <id>report</id>
                         <phase>test</phase>
                         <goals>
                             <goal>report</goal>
                         </goals>
                     </execution>
                 </executions>
             </plugin>

Reply:
would do when adding test case

Reply:
changed in 8692795